cowprotocol/accounting/rewards/mainnet/mainnet_dashboard_query_2510345.sql

```diff
@@ -0,0 +1,222 @@
+                    else price
+                end
+            ) from prices.usd
+            where
+                blockchain = 'ethereum'
+                and contract_address = 0xdef1ca1fb7fbcdc777520aa7f396b4e015f497ab
+                and date(minute) = cast('{{end_time}}' as timestamp) - interval '1' day
+        ) as cow_price,
+        (
+            select avg(price) from prices.usd
+            where
+                blockchain = 'ethereum'
+                and contract_address = 0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2
+                and date(minute) = cast('{{end_time}}' as timestamp) - interval '1' day
+        ) as eth_price
+),
+
+-- BEGIN QUOTE REWARDS
+order_quotes as (
+    select
+        order_uid,
+        quote_solver
+    from cow_protocol_ethereum.order_rewards
+    where
+        block_number > (select start_block from block_range)
+        and block_number <= (select end_block from block_range)
+),
+
+winning_quotes as (
+    select
+        oq.order_uid,
+        quote_solver as solver
+    from order_quotes as oq
+    inner join cow_protocol_ethereum.trades as t on oq.order_uid = t.order_uid and oq.quote_solver != 0x0000000000000000000000000000000000000000
+),
+
+quote_rewards as (
+    select
+        solver,
+        least({{quote_reward}}, {{quote_cap_eth}} * (select eth_price / cow_price from conversion_prices)) * count(*) as quote_reward
+    from winning_quotes group by solver
+),
+
+pre_results as (
+    select
+        pr.solver,
+        coalesce(reward_wei, 0) / pow(10, 18) as reward_eth,
+        coalesce(network_fee_wei, 0) / pow(10, 18) as network_fee_eth,
+        coalesce(execution_cost_wei, 0) / pow(10, 18) as execution_cost_eth,
+        coalesce(reward_wei, 0) / pow(10, 18) * (select eth_price / cow_price from conversion_prices) as reward_cow
+    from primary_rewards as pr left outer join fees_and_costs as fc on pr.solver = fc.solver
+),
+
+aggregate_results as (
+    select
+        solver,
+        -- payment_eth,
+        reward_eth as primary_reward_eth,
+        reward_cow as primary_reward_cow,
+        network_fee_eth,
+        execution_cost_eth
+    from pre_results
+),
+
+combined_data as (
+    select
+        coalesce(ar.solver, ss.solver, qr.solver) as solver,
+        -- coalesce(ar.solver, qr.solver) as solver,
+        -- payment_eth,
+        network_fee_eth,
+        execution_cost_eth,
+        primary_reward_eth,
+        primary_reward_cow,
+        coalesce(quote_reward, 0) as quote_reward,
+        coalesce(slippage, 0) as slippage_eth,
+        -- 0 as slippage_eth,
+        concat(
+            '<a href="https://dune.com/queries/2332678?SolverAddress=',
+            cast(ar.solver as varchar),
+            '&CTE_NAME=results_per_tx&StartTime={{start_time}}&endTime={{end_time}}&MinAbsoluteSlippageTolerance=0&RelativeSlippageTolerance=0&SignificantSlippageValue=0" target="_blank">link</a>'
+        ) as slippage_per_tx,
```

Comment:
This probably needs an update too.

Reply:
Actually it is used nowhere so it could just be removed.

Reply:
> This probably needs an update too.

Updated, where I assume this PR will actually be merged first
https://github.com/cowprotocol/dune-queries/pull/33/files

Reply:
I revied that PR so we could indeed merge it first.